requirements.txt

```diff
@@ -21,6 +21,7 @@ mock==2.0.0
 num2words==0.5.6
 ofxparse==0.19
 passlib==1.7.1
+pdf417gen==0.7.1
```

Comment:
so this line needs to be removed and I would need @d-fence to add it in the runbot image + document it in the l10n_cl_edi manifest.

Reply:
there is a bullet added in the description to l10n_cl_edi, explaining this.

Reply:
ok removed, but why don't define it in requirements?